src/components/CMessageTable.vue

```diff
@@ -64,9 +46,16 @@
             @onToggle="toggleRow"
           ></c-message-row>
           <tr :key="m.messageId + 0" v-if="openedRows.includes(m.messageId)">
-            <td class="border px-4 py-2" :colspan="columns.length">
-              <pre class="text-xs whitespace-normal ml-2 bg-white">Args: {{ m.args }}</pre>
-              <pre class="text-xs whitespace-normal ml-2 bg-white">Kwargs: {{ m.kwargs }}</pre>
+            <td class="border text-xs px-4 py-2" :colspan="6">
+              Args:
+              <p class="ml-2 bg-white">{{ m.args }}</p>
+              Kwargs:
+              <p class="ml-2 bg-white">{{ m.kwargs }}</p>
+            </td>
+            <td class="border px-4 py-2" :colspan="3">
```

Comment:
keep everything in one column and put queue_name in first (args and kwags can be very big)